siege_utilities/engines/dataframe_engine.py

```
@@ -192,6 +192,87 @@ def to_geodataframe(
+
+    def index_points(
+        self,
+        df: Any,
+        lat_col: str,
+        lon_col: str,
+        *,
+        grid: Optional[str] = None,
+        resolution: Optional[int] = None,
+        level: Optional[int] = None,
+        as_token: bool = True,
+    ) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `index_points`

(ANN401)